enos/modules/run_workloads/main.tf

```diff
@@ -52,14 +61,26 @@ resource "enos_local_exec" "get_allocs" {
   inline = ["nomad alloc status -json | jq '[.[] | select(.ClientStatus == \"running\")] | length'"]
 }
 
+resource "local_file" "vault_workload" {
+  filename = "${path.module}/jobs/vault-secrets.nomad.hcl"
```

Comment:
This is going to write the `vault-secrets.nomad.hcl` into a source controlled location, resulting a dirty git directory when running locally. Could we handle this by making the `template.data` a HCL [`file`](https://developer.hashicorp.com/nomad/docs/job-specification/hcl2/functions/file/file) function call? Then we could write the template with the secret path to somewhere like `/tmp/template.txt` and include it like 

```hcl
template {
  data = file("/tmp/template.txt")
}
```

Reply:
I updated it, but now we have a hardcoded relative [path: ](https://github.com/hashicorp/nomad/blob/65e76c633cfcdf854cfe738590494f80c4cec56f/enos/modules/run_workloads/jobs/vault-secrets.nomad.hcl#L57), cant we just add a .gitignore entry for the vault job spec?

Reply:
Yeah, that turns out to be kinda gross. Normally you'd handle this sort of problem with a HCL2 variable, but the way we're invoking the jobs doesn't make it very nice to have a one-off variable either. If you want to change it back and add a `.gitignore`, go ahead, but I'll approve the PR either way.